platform/podjs/src/browserPod.ts

```diff
@@ -483,10 +572,21 @@ function createUUID(): string {
 }
 
 let files = new Map<string, Stats>();
+
+/**
+ * The BrowserPolyNav @class implements the PolyNav interface.
```

Comment:
Well, that's pretty much exactly what `class BrowserPolyNav implements PolyNav` already says :D I'd suggest if we can't come up with something meaningful, we don't add doc comments.

Something more meaningful could be: "A PolyNav implementation that aims to feel as native as possible in the browser"